internal/provider/kubernetes/status.go

```diff
@@ -397,6 +398,41 @@ func (r *gatewayAPIReconciler) subscribeAndUpdateStatus(ctx context.Context) {
 		)
 		r.log.Info("envoyExtensionPolicy status subscriber shutting down")
 	}()
+
+	// EnvoyExtensionPolicy object status updater
+	go func() {
```

Comment:
can we wrap a `if ExtensionManager enabled` around this ?

Reply:
Done.